### ISSUE TITLE
[Admin] [New enterprise] Latitude and longitude are not automatically calculated

### ISSUE THREAD

**Issue by audez** [openfoodfoundation/openfoodnetwork#10006]
## Description
From the admin dashboard, when creating a new enterprise, entering the address alone (and saving it) will not automatically calculate latitude and longitude, leading to the enterprise not be placed on the map. The user must select 'calculate latitude and longitude' and only then the enterprises appears on the map.

It leads to a lot of enterprises not showing up on the general map. 

## Expected Behavior
Entering the address should calculate latitude and longitude, and place the enterprise on the map.
This would be coherent with the fact that when you create a shop from the “Open my shop” button (https://coopcircuits.fr/sell), it will automatically calculate the latitude and longitude even if you didn’t click “Locate address on the map”.

## Actual Behaviour
Entering the address alone doesn't place the enterprise on the map.

## Steps to Reproduce
1. Create a new enterprise (admin/enterprises/new)
2. Add an address, save
3. See that it's not located on the map
4. In this enterprise settings, in the Address tab (admin/enterprises/jdjdd/edit#!#address_panel), check the "Calculate latitude and longitude automatically from address?" box and save
5. See that the enterprise is now located on the map

### From admin dashboard: Create an enterprise and save with an address --> not on the map
![Screen Shot 2022-11-16 at 20 21 42](https://user-images.githubusercontent.com/28386648/202274877-a905a93d-1732-4422-b018-bf842d0e8203.png)
![Screen Shot 2022-11-16 at 20 21 36](https://user-images.githubusercontent.com/28386648/202274891-5a25fdc8-bda5-4a83-9f35-4a0231565822.png)

### From admin dashboard: Check the latitude and longitude box --> on the map
![Screen Shot 2022-11-16 at 20 24 31](https://user-images.githubusercontent.com/28386648/202274948-32336930-cac5-4424-b21b-169bc617bb28.png)
![Screen Shot 2022-11-16 at 20 22 17](https://user-images.githubusercontent.com/28386648/202274961-eb8c0ff4-98f1-4ac9-8235-3106a7895f15.png)

### From the “Open my shop” button: location is automatically calculated even if you didn’t click “Locate address on the map"
![Screen Shot 2022-11-12 at 14 06 56](https://user-images.githubusercontent.com/28386648/202275817-dc2764fe-8182-4599-a160-c7aa7a043bfd.png)

## Workaround
Check the box

## Severity
bug-s3: a feature is broken but there is a workaround

**Comment by RachL** [comment]
Yep, not a bug. It was designed this way. Moving to wishlist :+1:
